docker-for-mac/release-notes.md

```diff
@@ -16,11 +16,44 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
 {: .important }
 
+## Docker Desktop Community 3.2.0
+2021-03-NN
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### New
+
+- The dashboard shows a tip of the week.
+- The dashboard automatically opens when starting Docker Desktop.
+- Add experimental support to use the new Apple Virtualization framework (requires macOS Big Sur 11.1 or later)
+- Use iTerm2 to do `docker exec` if iTerm2 is installed, if not then use the default Terminal.App [docker/for-mac#4273](https://github.com/docker/for-mac/issues/4273)
+
+### Upgrades
+
+- [Docker Engine 20.10.4](https://docs.docker.com/engine/release-notes/#20104)
+- [Docker Compose 1.28.4](https://github.com/docker/compose/releases/tag/1.28.4)
+- [Compose CLI v1.0.8](https://github.com/docker/compose-cli/tree/v1.0.8)
+- [Docker Hub Tool v0.3.0](https://github.com/docker/hub-tool/releases/tag/v0.3.0)
+- [QEMU 5.0.1](https://wiki.qemu.org/ChangeLog/5.0)
+- [Amazon ECR Credential Helper v0.5.0](https://github.com/awslabs/amazon-ecr-credential-helper/releases/tag/v0.5.0)
+- [Alpine 3.13](https://alpinelinux.org/posts/Alpine-3.13.0-released.html)
+- [Kubernetes 1.19.7](https://github.com/kubernetes/kubernetes/releases/tag/v1.19.7)
+- [Go 1.16](https://golang.org/doc/go1.16)
+
+### Bug fixes and minor changes
+
+- Fixed an issue when port forwarding multiple ports with an IPv6 container network. Fixes [docker/for-mac#5247](https://github.com/docker/for-mac/issues/5247)
+- Fixed a regression where `docker load` could not use an xz archive anymore. Fixes [docker/for-mac#5271](https://github.com/docker/for-mac/issues/5271)
+- Fixed navigation issue between apps/containers screens [docker/for-win#10160](https://github.com/docker/for-win/issues/10160#issuecomment-764660660)
+- Fixed container instance view with long container/image name [docker/for-mac#5290](https://github.com/docker/for-mac/issues/5290)
+- Fixed an issue when binding ports on specific IPs. Notice: It may now take a bit of time before an inspect shows the open ports. Fixes [docker/for-mac#4541](https://github.com/docker/for-mac/issues/4541)
+- Fixed scrolling on container detail screen [docker/for-win#10160](https://github.com/docker/for-win/issues/10160)
```

Comment:
We can be more specific:

Fixed an issue on the container detail screen where the buttons would disappear when scrolling the logs.

Also let's move it higher up as this was the bug we got the most complaints about.

Reply:
There is a Mac ticket for this too, https://github.com/docker/for-mac/issues/5290

Reply:
Thanks, I'll update that.